preview/pages/modals.html

```diff
@@ -8,27 +8,60 @@
 
 <div class="card">
 	<div class="card-body">
-		<div class="btn-list">
-			{% include "ui/button.html" text="Simple modal" modal-id="simple" %}
-			{% include "ui/button.html" text="Large modal" modal-id="large" %}
-			{% include "ui/button.html" text="Small modal" modal-id="small" %}
-			{% include "ui/button.html" text="Full width modal" modal-id="full-width" %}
-			{% include "ui/button.html" text="Scrollable modal" modal-id="scrollable" %}
-			{% include "ui/button.html" text="Modal with form" modal-id="report" %}
-			{% include "ui/button.html" text="Success modal" modal-id="success" %}
-			{% include "ui/button.html" text="Danger modal" modal-id="danger" %}
-			{% include "ui/button.html" text="Modal with simple form" modal-id="team" %}
+		<div class="row g-5">
+	<div class="col-3">
```

Comment:
Removes the navigation on mobile:
```suggestion
	<div class="d-none d-md-block col-3">
```
Otherwise this happens
<img width="497" alt="Bildschirmfoto 2025-02-28 um 01 04 27" src="https://github.com/user-attachments/assets/73671d79-9722-4aaf-ac08-850de23110d4" />